### DISCUSSION TITLE
Is there a way to disable to kill ring in meow?

### DISCUSSION THREAD

**Discussion by chookity-pokk** [meow-edit/meow#474] [Q&A]
I disabled the kill ring in emacs with `(setq kill-ring nil)` but it doesn't seem to work with Meow. My situation is I have a line that has 12 spaces in it and I have something I want to copy and paste there (this is being done of about 20 positions in my script). What I want to do is erase the spaces by going on the line and hitting `s` which is kill then hitting `p` to paste my code. The problem is that is just gives the the 12 spaces.

What I want is to just never have the text I delete with `s` to show up in my killring.

**Comment by DogLooksGood** [comment]
To delete a region without touching the kill-ring, there's a built-in command `delete-region`(no bound by default).
To replace multiple consecutive spaces with one, you can use `M-SPC` (cycle-spacing, a DWIM command).

**Reply by chookity-pokk** [reply]
Could I just replace `meow-kill` with the `delete-region` command?

**Reply by DogLooksGood** [reply]
Yep, you can replace any command, there's no limitation. It's all up to your preference.
But then how you do cut / paste?

**Reply by chookity-pokk** [reply]
Personally, I don't know if I've ever really wanted to keep the things I cut in my killring. I personally just use `y` and then `p` for copy/pasting.
